Classes/AbstractDataHandlerListener.php

```diff
@@ -84,6 +84,39 @@ protected function getSubPageIds($pageId)
     }
 
     /**
+     * Checks if a page update will trigger a recursive update of pages
+     *
+     * This can either be the case if some $changedFields are part of the TriggerConfiguration or
```

Comment:
I think it'd be good to find a better / more precise name for `TriggerConfiguration`... What does it trigger? What does it configure?  
How about `RecursiveUpdateTriggerConfiguration`?